tools/test-backend

```diff
@@ -204,4 +246,4 @@ if __name__ == "__main__":
         print('FAILED!')
     else:
         print('DONE!')
-    sys.exit(bool(failures))
+    sys.exit(0)
```

Comment:
@timabbott When the former was used, the subprocess.check_call kept on throwing this error 
`  File "zulip/tools/test-backend", line 157, in <module>
    subprocess.check_call(cmd)
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['tools/test-backend', u'zerver.tests.test_signup.PublicURLTest.test_get_gcid_when_not_configured']' returned non-zero exit status 1
`
That's why I changed it.